support-frontend/assets/pages/new-contributions-landing/components/ContributionFormContainer.jsx

```diff
@@ -125,9 +125,9 @@ function campaignSpecificDetails() {
   if (isFrontlineCampaign()) {
     return {
       formMessage: (
-        <div className="frontline-campaign">
-          <div className="frontline-campaign__headline">Make a contribution</div>
-          <div className="frontline-campaign__body">to our dedicated series ‘The Frontline’</div>
+        <div>
+          <div className="form-message__headline">Make a contribution</div>
```

Comment:
I made these classes general - it's the only instance right now of a `formMessage`, so let's optimistically assume it's look like this for all future campaigns...